sw/device/lib/dif/dif_pinmux.h

```diff
@@ -0,0 +1,231 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_PINMUX_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_PINMUX_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+
+// TODO - should be autogenerated?
+#define DIF_PINMUX_MIO_NUM 32
+
+/**
+ * Pinmux peripheral input selects.
+ *
+ * These select which Pad Control MIO output is connected to the requested
+ * peripheral input (MIO output to pinmux input: MIO -> peripheral).
+ */
+typedef enum dif_pinmux_insel {
+  kDifPinmuxInselTieToZero = 0,
+  kDifPinmuxInselTieToOne = 1,
+  kDifPinmuxInselPadFirst = 2,
+  kDifPinmuxInselPadLast = kDifPinmuxInselTieToOne + DIF_PINMUX_MIO_NUM,
+} dif_pinmux_insel_t;
+#define DIF_PINMUX_INSEL_NUM (kDifPinmuxInselPadLast + 1)
```

Comment:
I'm not sure if it is a good idea to define this enum with gaps while the other enums below cover the entire range.

Reply:
This enum is no more.